js/imageeditor/shared/ImageEditor.svelte

```diff
@@ -249,6 +250,8 @@
 
 	let crop: ImageEditor;
 	let crop_zoom: ZoomTool;
+	export let can_undo = false;
```

Comment:
We use the history to determine 'empty' state in the parent, so this is exported.